memory_server.py

```
@@ -289,9 +287,18 @@ def get_recent_history(lanlan_name: str):
+async def get_memory(query: str, lanlan_name: str):
+    """语义记忆已退环境，返回空结果占位。"""
     lanlan_name = validate_lanlan_name(lanlan_name)
-    return await semantic_manager.query(query, lanlan_name)
+    from config.prompts_sys import MEMORY_RECALL_HEADER, MEMORY_RESULTS_HEADER
+    _lang = get_global_language()
+    return (
+        _loc(MEMORY_RECALL_HEADER, _lang).format(name=lanlan_name)
+        + query
+        + "\n\n"
+        + _loc(MEMORY_RESULTS_HEADER, _lang).format(name=lanlan_name)
+        + "\n（语义记忆已下线，暂无相关记忆片段。）"
```

Ruff (0.15.5)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)